lib/secure-boot-lib.robot

```diff
@@ -218,3 +218,26 @@ Execute File In UEFI Shell
     Press Enter
     ${out}=    Read From Terminal Until    FS0:\\>
     RETURN    ${out}
+
+Remove Old Secure Boot Keys
+    [Documentation]    Removes all files and directories in
```

Comment:
@m-iwanicki documentation section of those keywords could tell that we are using sbctl there